contracts/test/RiskManagerV1Stub.sol

```diff
@@ -32,4 +32,8 @@ contract RiskManagerV1Stub is RiskManagerV1 {
         tbtcToken.safeTransferFrom(msg.sender, address(this), amount);
         tbtcSurplus = tbtcSurplus.add(amount);
     }
+
+    function setOpenAuctionsCount(uint256 _openAuctionsCount) external {
```

Comment:
Is it used anywhere now?

Reply:
Nope. Removed it https://github.com/keep-network/coverage-pools/pull/88/commits/c5b872eb5156bbf7e2dfe8d16c496505fd046f9c